core/src/raw-base-cards.ts

```diff
@@ -1,41 +1,13 @@
-import StringSchemaJS from '!raw-loader!@cardstack/base-cards/string/schema.js';
-import StringEmbedded from '!raw-loader!@cardstack/base-cards/string/embedded.hbs';
-import DateSchemaJS from '!raw-loader!@cardstack/base-cards/date/schema.js';
-import DateEmbedded from '!raw-loader!@cardstack/base-cards/date/embedded.hbs';
-import TagSchemaJS from '!raw-loader!@cardstack/base-cards/tag/schema.js';
-import TagEmbedded from '!raw-loader!@cardstack/base-cards/tag/embedded.hbs';
-import CommentSchemaJS from '!raw-loader!@cardstack/base-cards/comment/schema.js';
-import CommentEmbedded from '!raw-loader!@cardstack/base-cards/comment/embedded.hbs';
+import StringCard from '!@cardstack/card-loader!@cardstack/base-cards/string/card.json';
+import DateCard from '!@cardstack/card-loader!@cardstack/base-cards/date/card.json';
+import TagCard from '!@cardstack/card-loader!@cardstack/base-cards/tag/card.json';
+import CommentCard from '!@cardstack/card-loader!@cardstack/base-cards/comment/card.json';
```

Comment:
I would have loved to just do
```
import CommentCard from '@cardstack/base-cards/comment'
```
But it wouldn't work because: 
1. The config approach wasn't picking up the card-loader
2. It was yelling that nothing was there since it's a directory. I wasn't sure if we could simulate a card.json when not there, or something along those lines?

Reply:
I think we want to go even further than that, like

```js
import everything from '!@cardstack/card-loader!@cardstack/base-cards/package.json'
```